worker/src/main/java/gov/cms/ab2d/worker/service/FileService.java

```diff
@@ -0,0 +1,49 @@
+package gov.cms.ab2d.worker.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.nio.file.StandardOpenOption.APPEND;
+
```

Comment:
All I/O related activities like creating a directory or file or writing to one has been extracted from the main business logic into this class, so that the single responsibility of this class is doing I-O.